src/javascript/app_2/App/Components/Elements/PositionsDrawer/result-overlay.jsx

```diff
@@ -60,6 +62,7 @@ const ResultOverlay = ({
 
 ResultOverlay.propTypes = {
     contract_id  : PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    display_shade: PropTypes.bool,
     onClick      : PropTypes.func,
     onClickRemove: PropTypes.func,
```

Comment:
FYI, the `close` or `X` button will still work but is not visible to users since it is part of the `overlay` which has `visibility: hidden;` when `shade` is not visible. Maybe we need to separate it out from the `overlay` component and make it `visible` once contract has ended.

Reply:
Working on it, Thanks for noticing that.